jolt-core/src/main/java/com/bazaarvoice/jolt/chainr/spec/ChainrEntry.java

```diff
@@ -48,10 +49,10 @@
     static {
         HashMap<String, String> temp = new HashMap<>();
         temp.put( "shift", Shiftr.class.getName() );
-        temp.put( "default", com.bazaarvoice.jolt.Defaultr.class.getName() );
-        temp.put( "overwrite-beta", Templatr.Overwritr.class.getName() );
-        temp.put( "default-beta", Templatr.Defaultr.class.getName() );
-        temp.put( "define-beta", Templatr.Definr.class.getName() );
+        temp.put( "default", Defaultr.class.getName() );
+        temp.put( "+modify-beta", Modifier.Overwritr.class.getName() );
+        temp.put( "~modify-beta", Modifier.Defaultr.class.getName() );
```

Comment:
The commit message for this change now no longer matches this.

Thinking this should be 
modify-overwrite-beta
modify-default-beta
modify-define-beta